app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AnswersController < ApplicationController
+  def answer
+    @question = Question.find(params[:question])
```

Comment:
1. You don't need instance variables anymore.
2. `is_*` is not the right name in the Ruby world (for both method names and variables). Also `right` feels a bit weird, how about just `correct`?
3. Names of your parameters are semantically incorrect. `question` -> `question_id`, `ans` -> `answer`, `test` -> `test_id`. Don't forget to apply corresponding changes to your form